compliance/azure/azure_disallowed_regions/README.md

```diff
@@ -1,48 +1,49 @@
+- When resource in disallowed region is detected, an email action is triggered automatically to notify the specified users of the incident. Users then have the option to terminate resources after manual approval if needed.
+- After approving the terminate action, all the resources may not get deleted due to the dependency.
+- So, if you find any error about dependency of the resources, those resources will get deleted in next schedule or you can re-execute the policy.
+- If there is any error about supported api-version for any of the resource then you need to manually delete the particular resource.
 
-### Azure Required Permissions
-- Microsoft.Resources/subscriptions/resources/read
-
-### Installation
-
-1. Follow steps to [Create an Azure Active Directory Application](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#create-an-azure-active-directory-application)
-1. Grant the Azure AD Application access to the necessary subscription(s)
-1. [Retrieve the Application ID & Authentication Key](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-application-id-and-authentication-key)
-1. Create RightScale Credentials with values that match the Application ID (Credential name: `AZURE_APPLICATION_ID`) & Authentication Key (Credential name: `AZURE_APPLICATION_KEY`)
-1. [Retrieve your Tenant ID](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-tenant-id)
-
-### Input Parameters
+## Input Parameters
 
 This policy has the following input parameters required when launching the policy.
 
-- *Azure AD Tenant ID* - the Azure AD Tenant ID used for the Azure API Authentication
-- *Azure Subscription ID* - the Azure Subscription ID used for the Azure API Authentication
 - *Approved Azure Region(s)* - list of approved Azure regions
 - *Email addresses of the recipients you wish to notify* - A list of email addresses to notify
 
-### Policy Actions
+## Prerequisites
+
+This policy uses [credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) for connecting to the cloud -- in order to apply this policy you must have a credential registered in the system that is compatible with this policy. If there are no credentials listed when you apply the policy, please contact your cloud admin and ask them to register a credential that is compatible with this policy. The information below should be consulted when creating the credential.
+
+### Credential configuration
+
+For administrators [creating and managing credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) to use with this policy, the following information is needed:
+
+Provider tag value to match this policy: `azure_rm`
+
+Required permissions in the provider:
+
+- Microsoft.Resources/subscriptions/resources/read
```

Comment:
I believe there are more permissions needed. At least the permission to delete the resources. Can you please add other permissions needed.

Reply:
Added permission to delete the resources